src/collective/volto/formsupport/restapi/services/submit_form/post.py

```diff
@@ -269,8 +277,30 @@ def send_data(self):
             if header_value:
                 msg[header] = header_value
 
+        should_send = self.block.get("send", [])
         self.manage_attachments(msg=msg)
-        self.send_mail(msg=msg, charset=charset)
+        if should_send:
```

Comment:
Could be better to make this check before preparing the message? If not should_send you can return without doing anything..or not?